docker/webui/nginx-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+replicas=""
+for i in $(seq 2 $replicas); do
+  replicas="server webui_webui_$i:9526;$replicas"
+done
+
+reg="s/REPLICAS/$replicas/"
+sed "$reg" /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
+
+cat /etc/nginx/conf.d/default.conf
+
+/docker-entrypoint.sh
+nginx
+
+sleep infinity
```

Comment:
```suggestion
nginx -g daemon off
```
See [NGINX Dockerfile](https://hub.docker.com/layers/nginx/library/nginx/latest/images/sha256-d73056bfea356376bf2464e544cc5508d8670f51b56b17a76638ee5433fb8e9f?context=explore)

Reply:
That's what I've meant with starting the server in daemon mode would be the preferred solution in my last review.

Reply:
Yes, I avoided that in the commit because this error

```nginx: invalid option: "off"```

I am going to check versions and alternatives

Reply:
Solved: nginx -g "daemon off;"